airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -24,6 +24,26 @@
 Changelog
 ---------
 
+4.4.0
+.....
+
+Features
+~~~~~~~~
+
+* ``feat(KubernetesPodOperator): Add support of container_security_context (#25530)``
+* ``Add @task.kubernetes taskflow decorator (#25663)``
+* ``pretty print KubernetesPodOperator rendered template env_vars (#25850)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Avoid calculating all elements when one item is needed (#26377)``
+* ``Apply PEP-563 (Postponed Evaluation of Annotations) to non-core airflow (#26289)``
```

Comment:
I think this one should be move down under the "Below changes" - it's not really user facing.

Reply:
Good shout